### DISCUSSION TITLE
Push Notification  request  permission  again

### DISCUSSION THREAD

**Discussion by skibaalex** [ionic-team/capacitor#3207] [Q&A] [answered]
is there a way to preform a request to allow push notifications after it was declined at the first time?

**Comment by jcesarmobile** [comment, marked as answer]
No, iOS only ask once for permissions (unless you delete the app).

**Reply by skibaalex** [reply]
i will mark that as the answer since i couldnt find a way but couldnt find anything on apples docs either 🤷‍♂️
